doc/user/layouts/shortcodes/fnlist.html

```diff
@@ -39,12 +38,28 @@ <h3 id="{{ lower .type | replaceRE "[^a-zA-Z0-9]+" "-" }}-func">
         {{ highlight .signature "clojure" "hl_inline=true" }}
       </p>
 
-      <p>{{ .description | $.Page.RenderString }}</p>
+      <p>
+        {{ .description | $.Page.RenderString }}
+
+        {{ if .url }}
+        <a href="{{ .url | relURL }}">(docs)</a>.
```

Comment:
@morsapaes I'm not much of a fan of this formatting. That's why I changed it! I think it's hard to see the "(docs)" link because it gets buried at the end of the sentence. I also think it's confusing, because you're _already_ looking at the documentation for the function. The link doesn't tell you that what it's linking to are vastly more detailed docs. And, lastly, to make the formatting work with this parenthetical, it means that the descriptions in sql_funcs.yml need to think about whether there is or is not a `url` parameter and avoid including a period when the `url` parameter is present. I guarantee that most people are not going to get that right when they add documentation for a new function. 

What didn't you like about the separate "Read more" link that I had originally?

Reply:
It just looked clunky, and wasn't consistent with how we used it in other sections. This page has a bunch of inconsistencies, though, so it might just be a case of doing another pass and tweaking the CSS a bit to make it look less bad.

> I guarantee that most people are not going to get that right when they add documentation for a new function.

And yeah, I agree. It's infrequent enough that we add new functions with dedicated reference documentation, though, and I like how `(docs)` is more space-efficient. But I'll use a bout of boredom to adapt it to your suggestion, since it's more tenable in the long-term.

Reply:
Sounds good. No particular attachment to "Read more" or whatever I had tried—just would love something a little more visible than "(docs)", and with less of a possibility of the formatting going wrong when someone's not looking at the preview.